src/compliance/compliance-requirements.service.ts

```diff
@@ -74,4 +79,79 @@ export class ComplianceRequirementsService {
       }
     );
   }
+
+  public async deleteRequirement(
+    ticker: string,
+    id: BigNumber,
+    params: TransactionBaseDto
+  ): ServiceReturn<Asset> {
+    const { signer, webhookUrl } = params;
+    const asset = await this.assetsService.findOne(ticker);
+
+    const { requirements } = await asset.compliance.requirements.get();
+
+    if (!requirements.find(requirement => requirement.id.eq(id))) {
+      throw new NotFoundException(`Requirement with id ${id} does not exist`);
+    }
+
+    return this.transactionsService.submit(
+      asset.compliance.requirements.remove,
+      { requirement: id } as unknown as RemoveAssetRequirementParams,
+      {
+        signer,
+        webhookUrl,
+      }
+    );
+  }
+
+  public async deleteRequirements(
+    ticker: string,
+    params: TransactionBaseDto
+  ): ServiceReturn<Asset> {
+    const { signer, webhookUrl } = params;
+    const asset = await this.assetsService.findOne(ticker);
+
+    return this.transactionsService.submit(asset.compliance.requirements.reset, undefined, {
+      signer,
+      webhookUrl,
+    });
+  }
```

Comment:
```suggestion
```

You can call `setRequirements` with empty args so this method isn't needed.

The SDK `reset` method is syntactic sugar of `set({requirements: []}, ...)` so we don't need to expose this.